Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.yml

```
@@ -91,6 +105,106 @@ script:
   script: ''
   type: python
   commands:
+  - name: argus-add-attachment
+    arguments:
+    - name: case_id
+      required: true
+      description: ID of Argus case
+    - name: file_id
+      required: true
+      description: ID of attachement
+    outputs:
+    - contextPath: Argus.Attachment.responseCode
+      description: API response metadata, response code of this request
```

```suggestion
      description: The response code of this request.
```

I Don't think we need the `API response metadata` prefix, we want the description to be as short as possible.
True for all outputs descriptions.
You can read more about outputs description here: https://xsoar.pan.dev/docs/documentation/documentation_tips#outputs

---

This has been approved before, and I do not have the time to change this now. I would appreciate if we could focus on the changes in this PR, unless there are critical issues.